tests/merch.test.ts

```diff
@@ -0,0 +1,100 @@
+import { ForbiddenError } from 'routing-controllers';
+import { UserAccessType } from '../types';
+import { DatabaseConnection, MerchFactory, UserFactory, PortalState } from './data';
+import { ControllerFactory } from './controllers';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('collection editing', () => {
+  test('Appropriate Permission Check for Getting Archived Items', async () => {
+    const conn = await DatabaseConnection.get();
```

Comment:
nit: test descriptions should describe some behavior that the test is asserting. 'Appropriate Permission Check for Getting Archived Items' doesn't really describe a behavior - maybe something like 'users cannot view archived collections, but admins can'?